recipes/systemc/2.3.3/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class SystemcConan(ConanFile):
+    name = "systemc_accellera"
+    version = "2.3.3"
+    description = """SystemC is a set of C++ classes and macros which provide
+                     an event-driven simulation interface."""
+    homepage = "https://www.accellera.org/"
+    url = "https://github.com/conan-io/conan-center-index "
+    license = "MIT"
+    topics = ("simulation", "modeling", "esl", "tlm")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "disable_async_updates": [True, False],
+        "disable_copyright_msg": [True, False],
+        "disable_virtual_bind": [True, False],
+        "enable_assertions": [True, False],
+        "enable_immediate_self_notifications": [True, False],
+        "enable_pthreads": [True, False]
+    }
+    default_options = {
+        "shared": True,
```

Comment:
```suggestion
        "shared": False,
        "fPIC": True,
```

Reply:
@vanwinkeljan 
Can you please change the default?

Reply:
hmm thought I did so, will update :)